process/smartContract/process.go

```diff
@@ -495,42 +486,49 @@ func (sc *scProcessor) DeploySmartContract(
 
 	err = sc.prepareSmartContractCall(tx, acntSnd)
 	if err != nil {
+		sc.setSilentError(err)
 		log.Debug("Transaction error", "error", err.Error())
 		return nil
 	}
 
 	vmInput, vmType, err := sc.createVMDeployInput(tx)
 	if err != nil {
+		sc.setSilentError(err)
 		log.Debug("Transaction error", "error", err.Error())
 		return nil
 	}
 
 	vm, err := sc.vmContainer.Get(vmType)
 	if err != nil {
+		sc.setSilentError(err)
 		log.Debug("VM error", "error", err.Error())
 		return nil
 	}
 
 	vmOutput, err := vm.RunSmartContractCreate(vmInput)
 	if err != nil {
+		sc.setSilentError(err)
 		log.Debug("VM error", "error", err.Error())
 		return nil
 	}
 
 	err = sc.accounts.SaveAccount(acntSnd)
 	if err != nil {
+		sc.setSilentError(err)
 		log.Debug("Save account error", "error", err.Error())
 		return nil
 	}
 
 	results, consumedFee, err := sc.processVMOutput(vmOutput, txHash, tx, acntSnd, vmInput.CallType)
 	if err != nil {
+		sc.setSilentError(err)
```

Comment:
these are no silent errors - why are they needed ?

Reply:
Were used for tests - moved logic to "testUtils".